.github/workflows/test.yaml

```diff
@@ -22,29 +22,28 @@ jobs:
     strategy:
       matrix:
         php-version:
-          - '7.2'
           - '7.3'
           - '7.4'
         dependencies: [highest]
         allowed_to_fail: [false]
         variant: [normal]
         include:
-          - php-version: '7.2'
+          - php-version: '7.3'
             dependencies: lowest
             allowed_to_fail: false
             variant: normal
           - php-version: '8.0'
             dependencies: highest
             allowed_to_fail: true
             variant: normal
-          - php-version: '7.3'
+          - php-version: '7.4'
             dependencies: highest
             allowed_to_fail: false
             variant: 'symfony/symfony:"4.4.*"'
-          - php-version: '7.3'
+          - php-version: '7.4'
             dependencies: highest
             allowed_to_fail: false
-            variant: 'sonata-project/admin-bundle:"3.*"'
+            variant: 'sonata-project/admin-bundle:"dev-master"'
```

Comment:
@VincentLanglet, I mean cases like this.

Reply:
This should be done in dev-kit.

Currently only real version are supported by dev-kit.
4.x-dev will be replaced by `4.x-dev.*` and it won't work, so some dev are need in dev-kit.

IMHO, we should better start by merging this PR which blocks dev-kit.
Then we can update dev-kit config to replace dev-master to 4.x-dev if we want to.